pyproject.toml

```diff
@@ -1,18 +1,27 @@
 [tool.poetry]
-name = "vyxal-crosshatch"
+name = "crosshatch"
 version = "0.1.0"
 description = "A pretty REPL for Vyxal."
-authors = ["Ginger < 75683114+GingerIndustries@users.noreply.github.com>"]
+authors = ["Ginger <75683114+GingerIndustries@users.noreply.github.com>"]
 
 [tool.poetry.dependencies]
 python = ">=3.10,<3.11"
-vyxal = "^2.18.5"
+vyxal = "^2.18"
 rich = "^13.2.0"
 
 [tool.poetry.dev-dependencies]
 debugpy = "^1.6.2"
```

Comment:
Use black, it's great. Among other things, black will add a blank line to the end of all your files.

Reply:
> lgtm, but could we have a screen capture? I'm too lazy to install this and test it out myself :P

Sure, here you go!
<img width="531" alt="Screenshot_20230124_053241" src="https://user-images.githubusercontent.com/75683114/214436045-1251e0e9-839a-4311-a57c-890a2ea285a3.png">



> Use black, it's great. Among other things, black will add a blank line to the end of all your files.

Can't be bothered :b
Maybe it's possible to set up a workflow to auto-Blacken the code? IDK.

Reply:
Yeah, Vyxal uses that pre-commit ci thing to automatically do it

Reply:
I love those colors btw. Any way to have different highlighting for named function definitions and that `code` (I assume that's parsed as 4 commands)?

Reply:
> I love those colors btw. Any way to have different highlighting for named function definitions and that `code` (I assume that's parsed as 4 commands)?

Probably, but you've just made me notice another bug: That `code` shouldn't be highlighted at all! Just the lambda and semicolon.